scripts/Dockerfile

```diff
@@ -26,11 +26,11 @@ RUN apt-get update && \
     # Removed unused parts to make a smaller Docker image:
     apt-get clean && \
     rm -rf /var/lib/apt/lists/* && \
-    cd /home/builder/lib/android-ndk/ && \
+    cd ${HOME}/lib/android-ndk/ && \
     rm -Rf sources/cxx-stl/system && \
-    cd /home/builder/lib/android-sdk/tools && \
+    cd ${HOME}/lib/android-sdk/tools && \
```

Comment:
Since in docker we use **own** user called builder with **default** settings, the home directory is exactly `/home/builder`.
Docker does not know anything about this user and therefore variable $HOME does not have any sense and breaks existing configuration.

Just tried to build docker image with your changes and it obviously failed:
```
Accept? (y/N): All SDK package licenses accepted

Warning: File /home/builder/.android/repositories.cfg could not be loaded.      
[=======================================] 100% Unzipping... platform-tools/systr
/bin/sh: 1: cd: can't cd to /root/lib/android-ndk/
The command '/bin/sh -c apt-get update &&     apt-get -yq upgrade &&     apt-get install -yq sudo &&     adduser --disabled-password --shell /bin/bash --gecos "" builder &&     echo "builder ALL=(root) NOPASSWD:ALL" > /etc/sudoers.d/builder &&     chmod 0440 /etc/sudoers.d/builder &&     chmod 0755 /tmp/*.sh &&     su - builder -c /tmp/setup-ubuntu.sh &&     su - builder -c /tmp/setup-android-sdk.sh &&     apt-get clean &&     rm -rf /var/lib/apt/lists/* &&     cd ${HOME}/lib/android-ndk/ &&     rm -Rf sources/cxx-stl/system &&     cd ${HOME}/lib/android-sdk/tools &&     rm -Rf emulator* lib* proguard templates' returned a non-zero code: 2
```